### PR TITLE
Added support for disable manually add members to specific Vo or group

### PR DESCRIPTION
**Pull request by HejdaJakub**
* Now Vo admin or Group admin is able to set new attribute for his Vo/group, which disabled manual member adding.
* If this attribute is set for Vo, then this configuration will be set also for all groups in this Vo.

### CODE REVIEW

File: apps/admin-gui/src/assets/i18n/en.json [hunk 1]
```diff
@@ -283,6 +283,7 @@
     "MEMBERS": {
       "TITLE": "Members",
       "ADD_MEMBER": "Add",
+      "ADD_MEMBER_DISABLED": "You cannot add members manually in this Vo.",
```
Comment (xkureck) [inline apps/admin-gui/src/assets/i18n/en.json:286]:
```suggestion
      "ADD_MEMBER_DISABLED": "You cannot add members manually into this Vo.",
```
probably better wording

File: apps/admin-gui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -513,6 +514,7 @@
       "TITLE": "Members",
       "ACTIONS_DISABLED": "Actions with members are disabled, because this group is filled with members from external source.",
       "ADD_MEMBER": "Add",
+      "ADD_MEMBER_DISABLED": "You cannot add members manually in this group.",
```
Comment (xkureck) [inline apps/admin-gui/src/assets/i18n/en.json:517]:
```suggestion
      "ADD_MEMBER_DISABLED": "You cannot add members manually into this group.",
```

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts [hunk 3]
```diff
@@ -91,6 +99,11 @@ export class VoMembersComponent implements OnInit {
           }
           this.loading = false;
         }, err=> this.loading = false);
+
+        this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+          this.blockManualMemberAdding = attrValue.value !== null;
+        });
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts:105]: please move this block of code "up"
we want to load the page when all the asynchronous operations are done

File: apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts [hunk 4]
```diff
@@ -99,6 +108,15 @@ export class GroupMembersComponent implements OnInit {
...
         }, err => this.loading = false);
+
+        this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+          this.blockManualMemberAdding = attrValue.value !== null;
+          if (this.blockManualMemberAdding !== true) {
+            this.attributesManager.getGroupAttributeByName(groupId, "urn:perun:group:attribute-def:def:blockManualMemberAdding").subscribe(groupAttrValue => {
+              this.blockManualMemberAdding = groupAttrValue.value !== null;
+            });
+          }
+        });
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts:119]: please move this block of code "up"
we want to load the page when all the asynchronous operations are done

**Review by xkureck — CHANGES_REQUESTED** [review]

File: apps/admin-gui/src/assets/i18n/en.json (hunk 1 above)
Comment (HejdaJakub) [reply apps/admin-gui/src/assets/i18n/en.json]: Edited.

File: apps/admin-gui/src/assets/i18n/en.json (hunk 2 above)
Comment (HejdaJakub) [reply apps/admin-gui/src/assets/i18n/en.json]: Edited.

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts (hunk 3 above)
Comment (HejdaJakub) [reply apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts]: This block of code has been moved.

File: apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts (hunk 4 above)
Comment (HejdaJakub) [reply apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts]: This block of code has been moved.

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts [hunk 5]
```diff
@@ -81,6 +89,10 @@ export class VoMembersComponent implements OnInit {
     this.route.parent.params.subscribe(parentParams => {
       const voId = parentParams['voId'];
 
+      this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+        this.blockManualMemberAdding = attrValue.value !== null;
+      });
+
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts:95]: I am really sorry, I think I didnt explain properly what I wanted. What I wanted is the code below. 
basically inserting the asynchrounous operation into the subscrible block - the reason is that we want to wait for end of one asynchronous operation and then start another, so we really know when we can actually load the page.
Please the same do in the group-members component

      this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
        this.blockManualMemberAdding = attrValue.value !== null;

        this.voService.getVoById(voId).subscribe(vo => {
        this.vo = vo;
        this.setAuthRights();
        this.memberMethodService.getMembersCount(this.vo.id).subscribe( count => {
          this.count = count;
          if(count < 400) {
            this.onListAll();
          }
          this.loading = false;
        });
      });

**Review by xkureck — CHANGES_REQUESTED** [review]

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts (hunk 5 above)
Comment (HejdaJakub) [reply apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-members/vo-members.component.ts]: Sorry, I didn't figure it out correctly. This implemetation should be better.

**Review by xkureck — APPROVED** [review]

**Pull request merged**